### ISSUE TITLE
NTR: decreased kidney weight

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4833]
**Preferred term label:**
decreased kidney weight

**Definition (free text, please give PubMed ID)**
Kidney weight which is relatively low.

**Parent term (use hpo.jax.org/app)**
HP_0012210 'Abnormal renal morphology'

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

@pnrobinson do you think we would want to add a term like this? this is for a Kids First dataset, the description is:
Left kidney 2.2 g, right kidney 1.5g (normal 4.2g +/- 0.7g)

**Comment by nicolevasilevsky** [comment]
Also need decreased spleen weight, the description is:
spleen with decreased weight for age 0.3g (normal 57 +/- 8 g)

**Comment by nicolevasilevsky** [comment]
@pnrobinson is it okay to add these terms?

**Comment by pnrobinson** [comment]
not sure if this is different from the hypoplasia term?

**Comment by nicolevasilevsky** [comment]
oh yes, good point! I'll use those terms, and close this ticket, thanks!
